Makefile

```diff
@@ -149,6 +151,14 @@ runner-build:
 		--cache-from $(CANONICAL_RUNNER_IMAGE):builder \
 		-t $(RUNNER_IMAGE):$(RUNNER_IMAGE_TAG_PREFIX)builder -f test/e2e/Dockerfile test/e2e
 
+	@echo "building target tests-1.33"
+	@docker build --target tests-1.33 \
+		--cache-from $(RUNNER_IMAGE):$(RUNNER_IMAGE_TAG_PREFIX)builder \
+		--cache-from $(CANONICAL_RUNNER_IMAGE):builder \
+		--cache-from $(RUNNER_IMAGE):$(RUNNER_IMAGE_TAG_PREFIX)tests-1.33 \
+		--cache-from $(CANONICAL_RUNNER_IMAGE):tests-1.33 \
+		-t $(RUNNER_IMAGE):$(RUNNER_IMAGE_TAG_PREFIX)tests-1.33 -f test/e2e/Dockerfile test/e2e
+
 	@echo "building target tests-1.32"
```

Comment:
`building target tests-1.32` and older versions need some edits: they sorta accumulate cache from the previously built (i.e. 1.32  should use cache from 1.33, 1.31 - cache from 1.32 and 1.33 etc.)